api/api.ts

```diff
@@ -0,0 +1,70 @@
+import axios from 'axios';
+import { GameSlug, UserInfo, UserGames } from 'types/responseInterface';
+
+const END_POINT = 'http://localhost:8080';
+
+const getRequest = async <T>(url: string): Promise<T | void> => {
+    try {
+        const response = (await axios.get(url)) as T;
+        console.log(response);
+        return response;
+    } catch (e) {
+        console.error(e.response);
+        // return e.response;
+    }
+};
+

```

Comment:
async로 감싼 이유가 있을까요?
따로 처리가 없다면 한번더 Promise로 감쌀 필요가 없을꺼 같아요.
또 END_POINT를 여기서 주입하는게 중복을 줄일수 있을꺼 같아요
```suggestion
const getRequest = <T>(url: string) => {
    try {
        return axios.get<T>(`${END_POINT}/${url}`));
    } catch (e) {
        console.error(e.response);
        // return e.response;
    }
};


```

Reply:
넵 감사합니다.

궁금한점이 있는데요, 정상적으로 api 호출이 일어나지 않아 `catch` 문으로 들어올 때, 보통의 경우에선 어떤 프로세스를 거치는지 궁금합니다.

가령, 아무것도 없이 저렇게 `console` 를 출력한다던가, 혹은 에러값 `e` 도 반환을 한다던가, 어떤게 좋은 방법인지 궁금합니다.